defi/src/api2/scripts/updateMetadataExtra.js

```diff
@@ -0,0 +1,26 @@
+const metadata = require('../metadata.json')
```

Comment:
why not combine this script with updateMetadata.ts? that way you could make sure they execute atomically